Rdmp.Core.Tests/Curation/ImportTests/ShareLoadMetadataTests.cs

```diff
@@ -58,6 +58,7 @@ public void GatherAndShare_LoadMetadata_WithCatalogue()
 
             Assert.That(lmd2.Name, Is.EqualTo(lmd1.Name));
             Assert.That(cata2.Name, Is.EqualTo(cata1.Name));
+            Assert.That(true, Is.EqualTo(true));
```

Comment:
Um ... what?

Reply:
Must be left over from a test stub, will fix